Cluster/How to configure APIM for Service Fabric managed cluster.md

```diff
@@ -0,0 +1,695 @@
+- [Azure API Management](https://learn.microsoft.com/azure/api-management/).
+- Azure Key vault with certificate.
+
+## Process
+
+1. First, use New-AzResourceGroup to create a resource group to host the virtual network. Run the following code to create a resource group named TestRG in the eastus Azure region.
+
+    ```powershell
+    $resourceGroupName = 'testrg'
+    $location = 'eastus'
+    New-AzResourceGroup -Name $resourceGroupName -location $location
+    ```
+
+1. Create a Network Security Group for APIM
+
+    ```powershell
+    $networkSecurityGroup = New-AzNetworkSecurityGroup -Name 'vnet-apim-nsg' -ResourceGroupName $resourceGroupName  -Location $location
+    ```
+
+1. Configure NSG rules for APIM (Management endpoint for Azure portal and PowerShell)
+
+    ```powershell
+    Add-AzNetworkSecurityRuleConfig -Name 'AllowManagementEndpoint' `
+      -NetworkSecurityGroup $networkSecurityGroup `
+      -Description 'Management endpoint for Azure portal and PowerShell' `
+      -Access Allow `
+      -Protocol Tcp `
+      -Direction Inbound `
+      -Priority 300 `
+      -SourceAddressPrefix ApiManagement `
+      -SourcePortRange * `
+      -DestinationAddressPrefix VirtualNetwork `
+      -DestinationPortRange 3443
+
+    ## Updates the network security group. ##
+    Set-AzNetworkSecurityGroup -NetworkSecurityGroup $networkSecurityGroup
+    ```
+
+    > Note: Create more rules as needed as per [https://learn.microsoft.com/azure/api-management/virtual-network-reference?tabs=stv2#required-ports](https://learn.microsoft.com/azure/api-management/virtual-network-reference?tabs=stv2#required-ports)
+
+1. Use New-AzVirtualNetwork to create a virtual network named VNet with IP address prefix 10.0.0.0/16 in the TestRG resource group and eastus 2 location.
+
+    ```powershell
+    $vnet = @{
+        Name = 'VNet'
+        ResourceGroupName = $resourceGroupName
+        Location = $location
+        AddressPrefix = '10.0.0.0/16'
+    }
+
+    $virtualNetwork = New-AzVirtualNetwork @vnet
+    ```
+
+1. Use Add-AzVirtualNetworkSubnetConfig to create a subnet configuration named default with address prefix 10.0.0.0/24.
+
+    ```powershell
+    $sfmcSubnet = @{
+        Name = 'sfmc'
+        VirtualNetwork = $virtualNetwork
+        AddressPrefix = '10.0.0.0/24'
+    }
+
+    $apimSubnet = @{
+        Name = 'apim'
+        VirtualNetwork = $virtualNetwork
+        AddressPrefix = '10.0.1.0/24'
+        NetworkSecurityGroup = $networkSecurityGroup
+    }
+
+    Add-AzVirtualNetworkSubnetConfig @sfmcSubnet
+    Add-AzVirtualNetworkSubnetConfig @apimSubnet
+    ```
+
+1. Then associate the subnets configuration to the virtual network with Set-AzVirtualNetwork.
+
+    ```powershell
+    $virtualNetwork | Set-AzVirtualNetwork
+    ```
+
+1. Prepare the steps for SFMC BYOVNET as per [Bring your own virtual network](https://learn.microsoft.com/azure/service-fabric/how-to-managed-cluster-networking#bring-your-own-virtual-network)
```

Comment:
Explanation for SFMC BYOVNET?

Reply:
worded acronym and added additional info under requirements